ci-operator/config/openshift/oadp-operator/.config.prowgen

```diff
@@ -0,0 +1 @@
+multi_arch: yes
```

Comment:
This is an experimental feature that we are planning to deprecate in the future. You don't have to worry about that. However, this is a configuration for the prowjob generation which it ensure that the job will run on the only cluster we have that holds arm64 nodes. 

You will still have to add the `multi_arch: true` on every of the images in the ci-operator configuration.

Reply:
thanks, will add the missing config!